lib/webdriveragent.js

```diff
@@ -25,6 +24,37 @@ const WDA_AGENT_PORT = 8100;
 const WDA_BASE_URL = 'http://localhost';
 const BUILD_TEST_DELAY = 1000;
 
+
+class NoSessionProxy extends JWProxy {
```

Comment:
I tried, but got circular dependency error as a result :(
I don't know the other way how to get rid of it except of putting the class together with main WDA class